src/NuGet.CommandLine/Commands/DownloadCommandBase.cs

```diff
@@ -27,9 +27,6 @@ public ICollection<string> Source
             get { return _sources; }
         }
 
-        [Option(typeof(NuGetCommand), "CommandFallbackSourceDescription")]
-        public ICollection<string> FallbackSource { get; } = new List<string>();
```

Comment:
Why make this change? This feature is used (via NuGet.CommandLine.Cmd) for restores in this repo and VS Extension.